azurerm/internal/services/mssql/mssql_server_resource.go

```diff
@@ -136,6 +137,16 @@ func resourceArmMsSqlServer() *schema.Resource {
 				},
 			},
 
+			"minimal_tls_version": {
```

Comment:
This might be better as `minimum_tls_version`? or even `ssl_minimum_tls_version` ?

Reply:
Agreed I wouldn't have chosen `minimal` but it seems to have pervaded. It did prompt me to check the dictionary though and _technically_ it seems appropriate, though it feels a bit arcane to me.

Reply:
Yep, regardless of the api/sdk and existing properties i'd prefer we correct it (and the others) to minimum as technically right or not, to match what other resources/apis and the rest of the world uses